airflow/hooks/presto_hook.py

```diff
@@ -20,11 +20,13 @@ class PrestoHook(BaseHook):
     >>> ph.get_records(sql)
```

Comment:
ah. word. sorry for committing with errors... I've actually had trouble getting my tests to all run properly... the hive ones seem to always fail -- any advice?